lib/rucio/tests/test_did.py

```diff
@@ -1117,3 +1133,48 @@ def test_bulk_get_meta(self):
             list_dids.append({'scope': scope, 'name': cnt[idx]})
         list_meta = [_ for _ in self.did_client.get_metadata_bulk(list_dids)]
         assert len(list_meta) == 0
+
+    @pytest.mark.dirty
+    @pytest.mark.noparallel(reason='uses pre-defined RSE')
+    def test_bulk_get_meta_inheritance(self):
```

Comment:
it would be great if you could create new tests with generated RSEs from e. g. `rse_factory.make_rse()` to avoid noparallel and dirty markers.